plugin/trino-queryeventlistener/src/main/java/io/trino/plugin/queryeventlistener/QueryEventListener.java

```diff
@@ -26,48 +32,66 @@
         implements EventListener
 {
     private static final Logger logger = Logger.get(QueryEventListener.class);
+    static final ObjectMapper objectMapper = configuredObjectMapper();
 
     public QueryEventListener(Map<String, String> config)
     {
     }
 
+    private static ObjectMapper configuredObjectMapper()
```

Comment:
I usually like to put 'private static' (util) methods at the very bottom of the file, just because they're not what the class is "all about". 😄